client/disputes/index.tsx

```diff
@@ -345,9 +345,9 @@ export const DisputesList = (): JSX.Element => {
 	const downloadable = !! rows.length;
 
 	const onDownload = async () => {
-		recordEvent( 'wcpay_disputes_download', {
-			exported_disputes: rows.length,
-			total_disputes: disputesSummary.count,
+		recordEvent( 'wcpay_csv_export_click', {
+			location: 'disputes',
+			exported_rows: disputesSummary.count,
```

Comment:
`exported_rows` is clear 🎉 

Are there any other props that we should include here? e.g. from previous event(s) or other relevant context that might be useful to capture.